planetarium/NineChronicles
# Introduce `ResourceTestScene`

### Description

1. Implement `ResourceTestScene` which can test various art resources.

### How to test

1. Enter spine resource id and check if it's loading properly.
2. Enter background resource id and check if it's loading properly.

### Required Reviewers
@planetarium/9c-dev @caesty 

### Related Links
https://app.asana.com/0/958521740385861/1201200786778989

### Screenshot
![GIF 2022-01-20 오후 3-49-08](https://user-images.githubusercontent.com/26648527/150288323-964b45f4-3c22-48d1-8431-5d10fa70b124.gif)
![GIF 2022-01-20 오후 4-01-29](https://user-images.githubusercontent.com/26648527/150289450-09c6837c-9560-461e-b894-a1068cf95f91.gif)
